Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:16.04
+
+MAINTAINER "ROS Industrial" "https://github.com/ros-industrial"
+
+# Install packages
+RUN apt-get update -qq
+RUN apt-get -qq install -y --force-yes git sudo lsb-release python-pip wget
+RUN apt-get clean
```

Comment:
For various reasons it's recommended to use a single `RUN` for `apt-get`, see [Best practices for writing Dockerfiles - The Dockerfile instructions - apt-get](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get).